swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -189,14 +193,15 @@ class Adapter: @unchecked Sendable {
   deinit {
     Log.log("Adapter.deinit")
 
+    // Cancel network monitor
```

Comment:
This was moved to be first to ensure no new path updates are being triggered while the rest of the deinit block is being executed.